fastly/wasm.go

```diff
@@ -0,0 +1,96 @@
+package fastly
+
+import (
+	"fmt"
+	"io"
+	"time"
+)
+
+type WasmPackageMetadata struct {
+	Name        string
+	Description string
+	Authors     []string
+	Language    string
+	Size        int
```

Comment:
We should consider making this an `int64` to be future proof. I can check internally.